src/shared/utils/formatMeasuresHistory.ts

```diff
@@ -0,0 +1,82 @@
+import { format } from 'date-fns';
+
+import { MeasuresHistory } from '@types/project';
+
+const indexOfLongestArray = (measuresHistory: MeasuresHistory) => measuresHistory.results.reduce((prevIndex, historyResult, currentIndex) => {
+    const previousMetricHistoryLength = measuresHistory.results[prevIndex].collected_metric_history.length;
```

Comment:
olha como fica esse valor ;-; não acredita que ficaria melhor...

```suggestion
    const previousMetricHistoryLength = measuresHistory.results[prevIndex].collectedMetricHistory.length;
```

Reply:
Concordo, mas esse é o formato que vem da API